app.rb

```diff
@@ -0,0 +1,199 @@
+require_relative './classes/book'
+require_relative './classes/lable'
+require_relative './item'
+require_relative './classes/game'
+require_relative './classes/author'
+require_relative './classes/genre'
+require_relative './classes/musicalbum'
+require_relative './iofile/read_data'
+require_relative './iofile/save_data'
+require 'json'
+
+class App
+  def initialize
+    @books = []
+    @lables = []
+    @games = ReadData.new.read_games
+    @authors = []
+    @genres = []
+    list_all_stored_books
+    list_all_stored_lables
+    @musicalbums = ReadData.new.read_music_album

```

Comment:
- [ ] After exiting the console, only games and music albums, get preserved. Kindly ensure that all data are preserved including books

![image](https://user-images.githubusercontent.com/63473584/216719583-50e550f5-0f17-4ebe-ab63-8ed4fd3f3332.png)